modules/rosa-policy-change-management.adoc

```diff
@@ -232,4 +232,8 @@ creation.
 |- Implement change management best practices for customer
 applications and data hosted on the AWS Cloud.
 
-|===
\ No newline at end of file
+|===
+
+^[1]^ For more information on Authentication flow for AWS STS, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/authentication_and_authorization/managing-cloud-provider-credentials#cco-short-term-creds-auth-flow-aws-diagram_cco-short-term-creds[Authentication flow for AWS STS].
```

Comment:
Hey, I didn't see a "peer review done" label for this one, so I took a quick look - I won't hold up merging for this but I think this shouldn't be capitalized.

```suggestion
^[1]^ For more information on authentication flow for AWS STS, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/authentication_and_authorization/managing-cloud-provider-credentials#cco-short-term-creds-auth-flow-aws-diagram_cco-short-term-creds[Authentication flow for AWS STS].
```